source/docs/use/configuration/basic.md

```diff
@@ -1,84 +1,58 @@
 ---
-id: basic
-title: Initial Configuration
-description: 'Basic configuration'
+title: Initial configuration
+description: 'Setting up and authenticating the HACS integration'
 ---
 
-!!! info
-    _This page assumes that you have already completed the [download](/docs/use/download/prerequisites.md)._
+## Prerequisites
```

Comment:
Is this header still needed?

Reply:
it makes the section show in the sidebar